core/src/test/java/uk/gov/gchq/koryphe/tuple/ReflectiveTupleTest.java

```diff
@@ -145,6 +160,17 @@ public void shouldPutMethod() throws Exception {
         assertEquals("changed", record.fieldB);
     }
 
+    @Test
+    public void shouldPutMethodWithDot() throws Exception {
+        //given
+        final ExampleObj3 record = new ExampleObj3();
+        testObj = new ReflectiveTuple(record);
+        //when
+        testObj.put("fieldB.", "changed");
+        //then
+        assertEquals("changed", record.fieldB);
+    }
+
```

Comment:
Is this testing a plausible reference string? if so it errors. 
```
@Test
    public void shouldFindPublicFieldWithDotAtStart() throws Exception {
        assertEquals("fa", testObj.get("." + FIELD_A));
    }
```

or "nestedField..fieldA"